tox.ini

```diff
@@ -4,8 +4,11 @@ skip_missing_interpreters = True
 
 [testenv]
 install_command = pip install --process-dependency-links {opts} {packages}
```

Comment:
You can probably also remove this and use default value (`pip install {opts} {packages}`).